pep-0505.txt

```diff
@@ -504,10 +526,12 @@ Usage Of ``None`` In The Standard Library
 
 The previous sections show some code patterns that are claimed to be "common",
 but how common are they? The attached script `find-pep505.py
-<https://github.com/python/peps/blob/master/pep-0505/find-pep505.py>`_ is meant to
+<https://hg.python.org/peps/file/tip/pep-0505/find-pep505.py>`_ is meant to
```

Comment:
Sorry, I didn't realize that somebody rewrote this link when I submitted it to the peps e-mail list last year. Fixed in latest commit.